specification/cognitiveservices/resource-manager/readme.md

```diff
@@ -30,10 +30,55 @@ These are the global settings for the CognitiveServices API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2024-10
+tag: package-preview-2025-04-01
 ```
 
 
+### Tag: package-preview-2025-04-01-preview
+
+These settings apply only when `--tag=package-preview-2025-04-01-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2025-04-01-preview'
```

Comment:
this name is not the same as the tag you set in line 33.

Reply:
Thanks, the tag are generated using ARM provided script, so I didn't notice this issue, thanks for help checking